spec/lib/facilities/local_store_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'facilities/local_store'
+
+RSpec.describe Facilities::LocalStore do
+  let(:url) { 'https://www.example.com/VBA_Facilities' }
+  let(:bulk_client_stub) { instance_double('Facilities::BulkClient') }
+  let(:adapter_stub) { class_double('VBAFacilityAdapter') }
+  subject { described_class.new(url, nil, adapter_stub) }
+
+  before(:each) do
+    subject.instance_variable_set(:@client, bulk_client_stub)
```

Comment:
same comment as before -- i don't typically see this in specs, i would use mock object or stubbing to achieve this effect vs instance_variable_set.